cmdconfig/viper.go

```diff
@@ -28,14 +28,17 @@ func InitViper(v *viper.Viper) {
 			utils.FailOnError(err)
 		}
 	}
+
+	// set defaults
+	v.Set("show-spinner", true)
```

Comment:
so we ONLY disable spinner for non interactive queries?

Reply:
yes.

otherwise for things like `service start` and `service refresh`, the spinner will stop working